README.zh.md

```diff
@@ -0,0 +1,214 @@
+<img alt="Stateful Functions" src="docs/fig/stateful_functions_logo.png" width=350px/>
+
+Stateful Functions（简称 StateFun）是一个 [Apache Flink](https://flink.apache.org/) 库， __可简化构建分布式有状态应用程序的过程__ 。它基于可持久化状态的函数，这些函数可以在强一致性保证下进行动态交互。
+
+Stateful Functions 使我们能够将强大的状态管理与像 AWS Lambda 类似的 FaaS 实现和 Kubernetes 等现代资源编排框架的弹性、快速扩缩容和滚动升级功能相结合。通过这些特性，它解决了当今许多 FaaS 解决方案中 [最常被引用的两个缺点](https://www2.eecs.berkeley.edu/Pubs/TechRpts/2019/EECS-2019-3.pdf) ：函数间的状态一致性和高效消息传递。
+
+本文档旨在简要介绍 Stateful Functions 的核心概念以及如何开发一个 Stateful Functions 应用。
+
+更多详细信息，可以参考 [官方文档](https://ci.apache.org/projects/flink/flink-statefun-docs-master) ，相关代码示例，请查看[这里](statefun-examples/)  。
+
+[![构建状态](https://travis-ci.org/apache/flink-statefun.svg?branch=master)](https://travis-ci.org/apache/flink-statefun)
+
+## 目录
+
+- [核心概念](#core-concepts)
+   * [摘要](#abstraction)
+   * [函数模块和可扩展性](#modules)
+   * [Runtime](#runtime)
+- [入门](#getting-started)
+   * [运行一个完整的例子](#greeter)
+   * [创建项目](#project-setup)
+   * [构建项目](#build)
+   * [在 IDE 中运行](#ide-harness)
+- [应用部署](#deploying)
+   * [使用 Docker 映像进行部署](#docker)
+   * [作为 Flink 作业部署](#flink)
+- [参与贡献](#contributing)
+- [开源软件许可](#license)
+
+## <a name="core-concepts"></a>核心概念
+
+### <a name="abstraction"></a>摘要
+
+Stateful Functions 应用程序由以下原语组成：有状态函数，入口(ingresses)，路由器(routers)和出口(egresses)。
```

Comment:
```suggestion
Stateful Functions 应用程序由以下原语组成：有状态函数，入口(ingresses)，路由(routers)和出口(egresses)。
```
Refer to https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications